src/main/java/dgen/benchmark.java

```diff
@@ -0,0 +1,48 @@
+package dgen;
+
+import dgen.dataset.Dataset;
+import dgen.dataset.DatasetConfig;
+import dgen.dataset.DatasetGenerator;
+import dgen.utils.parsers.SpecificationParser;
+import dgen.utils.serialization.CSVSerializer;
+//import dgen.utils.serialization.ParquetSerializer;
+import dgen.utils.serialization.ParquetSerializer;
+import dgen.utils.serialization.PostgresSerializer;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+
+public class benchmark {
```

Comment:
benchmark -> Benchmark

Reply:
Whoops I keep accidentally pushing this file but it was mainly a junk file for me to run tests with. Removed in a507420a5460e77eef93fc4e7d6b1e0b559444b7